src/constants/announcementsList.js

```diff
@@ -0,0 +1,166 @@
+import imageA from "../assets/image/encourages/dog.jpg";
```

Comment:
Why image `dog` evolve into `imageA`?

Reply:
i forgot to rename it when i sent it to github